tests/api_test.py

```diff
@@ -80,6 +85,59 @@ def test_pytket_qir() -> None:
     )
 
 
+def test_pytket_qir_optimised() -> None:
+    circ = Circuit(3)
+    circ.H(0)
+
+    result = pytket_to_qir(
+        circ,
+        name="test_pytket_qir",
+        returntype=ReturnTypeQIR.STRING,
+        pyqir_0_7_compatibility=True,
+    )
+
+    print(result)
```

Comment:
```suggestion
```

Reply:
Do we want to remove all the prints of the result? They are currently in all of the testcases and I think it is somehow helpful to have them at the moment in case of any more changes?

Reply:
At some point we should, but if they're helpful at the moment we can keep them.